AnnService/inc/Core/SPANN/ExtraFullGraphSearcher.h

```diff
@@ -581,8 +778,8 @@ namespace SPTAG
                     }
 
                     m_listInfos[i].listOffset = (static_cast<uint64_t>(m_listPageOffset + pageNum) << PageSizeEx);
-                    m_listInfos[i].listEleCount = min(m_listInfos[i].listEleCount, (min(static_cast<int>(m_listInfos[i].listPageCount), p_postingPageLimit) << PageSizeEx) / m_vectorInfoSize);
-                    m_listInfos[i].listPageCount = static_cast<std::uint16_t>(ceil((m_vectorInfoSize * m_listInfos[i].listEleCount + m_listInfos[i].pageOffset) * 1.0 / (1 << PageSizeEx)));
+                    // m_listInfos[i].listEleCount = min(m_listInfos[i].listEleCount, (min(static_cast<int>(m_listInfos[i].listPageCount), p_postingPageLimit) << PageSizeEx) / m_vectorInfoSize);
```

Comment:
If it is not needed anymore, we can remove it instead of comment out

Reply:
> If it is not needed anymore, we can remove it instead of comment out

I'm not sure if these two lines are useful under no-compression setting, @MaggieQi can we just remove these two lines?

Reply:
Added a codebranch to keep these 2 lines for no-compression setting.